securityconfig/roles.yml

```diff
@@ -61,3 +61,35 @@ anomaly_full_access:
         - 'indices_monitor'
         - 'indices:admin/aliases/get'
         - 'indices:admin/mappings/get'
+
+# Allows users to read and download Reports
+reports_instances_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/reports/instance/list'
```

Comment:
Whats the difference between instance `list` and `get`?

Reply:
get -> get one item
list -> get all paginated response for items